Lib/test/test_code.py

```diff
@@ -337,6 +337,37 @@ def func():
         new_code = code = func.__code__.replace(co_linetable=b'')
         self.assertEqual(list(new_code.co_lines()), [])
 
+    def test_invalid_bytecode(self):
+        def foo(): pass
+        co = foo.__code__
+        CodeType = type(co)
+
+        new_co = CodeType(
```

Comment:
Please don't use the CodeType constructor, it's a pain to maintain. I would suggest to use foo.__code__.replace(co_code=...) instead. Maybe something like:

```
code = b'\xEE' + foo.__code__
foo.__code__.replace(co_code=code)
```

Reply:
I wasn't aware of this method, that's indeed much pleasant to the eye!